crates/nu-std/lib/help.nu

```diff
@@ -494,190 +500,195 @@ export def "help operators" [
-            print-help-header "Parameters"
-            for positional in $positionals {
-                print -n "  "
-                if ($positional.is_optional) {
-                    print -n "(optional) "
-                }
-                print $"(ansi teal)($positional.parameter_name)(ansi reset) <(ansi light_blue)($positional.syntax_shape)(ansi reset)>: ($positional.description)"
-            }
-
-            if $is_rest {
-                let rest = ($parameters | where parameter_type == rest | get 0)
-                print $"  ...(ansi teal)rest(ansi reset) <(ansi light_blue)($rest.syntax_shape)(ansi reset)>: ($rest.description)"
-            }
+            print-help-header -n "Category"
+            print $" ($command.category)"
         }
-    }
 
-    if not ($command.examples | is-empty) {
         print ""
-        print-help-header -n "Examples"
-        for example in $command.examples {
-            print ""
-            print $"  ($example.description)"
-            print $"  > ($example.example | nu-highlight)"
-            if not ($example.result | is-empty) {
-                for line in (
-                    $example.result | table | if ($example.result | describe) == "binary" { str join } else { lines }
-                ) {
-                    print $"  ($line)"
-                }
-            }
+        print "This command:"
+        []
+        if ($command.creates_scope) {
+            print $"- (ansi cyan)does create(ansi reset) a scope."
+        } else {
+            print $"- (ansi cyan)does not create(ansi reset) a scope."
+        }
+        if ($command.is_builtin) {
+            print $"- (ansi cyan)is(ansi reset) a built-in command."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a built-in command."
+        }
+        if ($command.is_sub) {
+            print $"- (ansi cyan)is(ansi reset) a subcommand."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a subcommand."
+        }
+        if ($command.is_plugin) {
+            print $"- (ansi cyan)is part(ansi reset) of a plugin."
+        } else {
+            print $"- (ansi cyan)is not part(ansi reset) of a plugin."
+        }
+        if ($command.is_custom) {
+            print $"- (ansi cyan)is(ansi reset) a custom command."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a custom command."
+        }
+        if ($command.is_keyword) {
+            print $"- (ansi cyan)is(ansi reset) a keyword."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a keyword."
         }
-    }
 
-    print ""
+        print ""
+    }
 }
 
-# Show help on nushell commands.
+# Show help on nushell commands. 
 export def "help commands" [
     ...command: string@"nu-complete list-commands"  # the name of command to get help on
     --find (-f): string  # string to find in command names and usage
+    --all (-a) # if showing help for single command, display all sections
```

Comment:
I see that you mentioned this in the PR description, but I think that `--verbose`  is more intuitive. Something like `help -a` makes me think that I'll get the help for all commands.

Reply:
For me, `--verbose` connotes extra output for debugging when the command is misbehaving.  How about `--full`?  Whatever we call it, are you OK with the default being less rather than more sections?

Reply:
I don't think `--verbose` is tied to debugging only. From my experience it represents a flag saying "give me the entire thing".

The thing with `--full` is that then the short flag would conflict with `--find`. 

And yes, I think that the default should be the shortened version.

Reply:
That being said, I think it would be a good idea to discuss this with more people in the Discord channel :)